src/webapp/components/DeliveryTable.js

```diff
@@ -1,79 +1,126 @@
+        table.scrollTop = row.offsetTop;
+      }
+    }
+  });
 
   // sort happens in-place
-  rows.sort((rowA, rowB) => rowB.timestamp - rowA.timestamp);
+  rows.sort((rowA, rowB) => rowA.timestamp - rowB.timestamp);
 
-  // format data for presentation
   const formattedRows = rows.map((row) => {
-    const timestamp = new Date(row.timestamp * 1000);
+    const isFocused = row.Code === focusedRequestId;
 
     return {
       ...row,
-      timestamp: `${timestamp.toLocaleDateString()}: ${timestamp.toLocaleTimeString()}`,
+      isFocused,
     };
   });
 
   return (
-    <TableContainer className={classes.container} component={Paper}>
+    <TableContainer
+      id="table-wrapper"
+      className={classes.container}
+      component={Paper}
+    >
       <Table aria-label="simple table">
         <TableHead>
           <TableRow>
-            <TableCell>{str("webapp:zoneFinder.label.code")}</TableCell>
-            <TableCell align="right">
-              {str("webapp:zoneFinder.label.crossStreetFirst")}
+            <TableCell>
+              {str("webapp:deliveryNeeded.table.headers.daysOpen", {
+                defaultValue: "Days open",
+              })}
             </TableCell>
-            <TableCell align="right">
-              {str("webapp:zoneFinder.label.crossStreetSecond")}
-            </TableCell>
-            <TableCell align="right">
-              {str("webapp:zoneFinder.label.firstName")}
-            </TableCell>
-            <TableCell align="right">
-              {str("webapp:zoneFinder.label.slackLink")}
-            </TableCell>
-            <TableCell align="right">
-              {str("webapp:zoneFinder.label.timestamp")}
+            <TableCell>{str("webapp:zoneFinder.label.firstName")}</TableCell>
+            <TableCell>{str("webapp:zoneFinder.label.code")}</TableCell>
+            <TableCell>
+              {str("webapp:deliveryNeeded.table.headers.crossStreets", {
+                defaultValue: "Cross streets",
+              })}
             </TableCell>
           </TableRow>
         </TableHead>
         <TableBody>
           {formattedRows.map((row) => (
-            <TableRow key={row.Code}>
-              <TableCell component="th" scope="row">
-                {row.Code}
+            <TableRow
+              id={row.Code}
+              key={row.Code}
+              className={row.isFocused ? classes.focused : ""}
+              onClick={() => setFocusedRequestId(row.Code)}
+            >
+              <TableCell>
+                <DaysOpenChip
+                  timeOnly
+                  daysOpen={daysSinceSlackMessage(row.slackTs)}
+                />
+              </TableCell>
+              <TableCell>
+                {row["First Name"]}
+                <Box className={classes.chipRow}>
+                  {row["For Driving Clusters"] && <DrivingClusterChip />}
+                  <HouseholdSizeChip size={row["Household Size"]} />
```

Comment:
should we only show the household chip if it has a size value other than n/a?

Reply:
good q. im displaying n/a just to keep it obvious that this data wasn't available as this is what is displayed on the Slack post